.github/CONTRIBUTING.md

```diff
@@ -54,16 +54,20 @@ Once the pull request is approved, a team member will take care of merging.
 
 ## Setup environment
 
-Setting up your KerasCV development environment requires you to fork the KerasCV repository,
-clone the repository, install dependencies, and execute `python setup.py develop`.
+Python 3.7 or later is required.
+
+Setting up your KerasNLP development environment requires you to fork the
+KerasNLP repository, clone the repository, create a virtual environment, and 
+install dependencies.
 
 You can achieve this by running the following commands:
 
 ```shell
 gh repo fork keras-team/keras-cv --clone --remote
 cd keras-cv
-pip install ".[tests]"
-python setup.py develop
+python -m venv ~/keras-cv-venv
```

Comment:
I have a question. I this required when we work with our [devcontainer/codespace](https://github.com/keras-team/keras-cv/tree/master/.devcontainer)? If not I suppose that we could add a notice

Reply:
I suppose Docker is completely independent of python virtual env. I didn't understand what kind of notice?

Reply:
Oh I did not remember that we still don't have an section to use `[devcontainer/codespace](https://github.com/keras-team/keras-cv/tree/master/.devcontainer)`. 
Probably we will add this later or we can open a separate ticket to track this.

Reply:
Yeah 👍